python/mxnet/gluon/block.py

```diff
@@ -1059,7 +1059,7 @@ def _call_cached_op(self, *args):
             out = [out]
         return _regroup(out, self._out_format)
 
-    def optimize_for(self, x, *args, backend=None, backend_opts=None, clear=True, **kwargs):
+    def optimize_for(self, x, *args, backend=None, clear=False, static_alloc=False, static_shape=False, **kwargs):
```

Comment:
dont we need to add the other hybridize args like: inline_limit=2, forward_bulk_size=None, backward_bulk_size=None

Reply:
I thought that this would clutter the args. 
inline_limit=2, forward_bulk_size=None, backward_bulk_size=None being rarely use, I think it's fine to let the user call hybridize separately if he needs them.